subprojects/docs/src/samples/testing/jacoco/quickstart/kotlin/build.gradle.kts

```
@@ -65,3 +65,23 @@ tasks.jacocoTestCoverageVerification {
     }
 }
 // end::violation-rules-configuration[]
+
+// tag::testtask-configuration-defaults[]
+tasks.getByName<Test>("test") {
```

:x: should use the accessor to the `test` task
```suggestion
tasks.test {
```